### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by GyyEcho** [1Panel-dev/1Panel#5949] [Q&A]
### 联系方式

liiecho@126.com

### 1Panel 版本

v1.10.13-lts

### 问题描述

![image](https://github.com/user-attachments/assets/579733a9-25fa-46bb-aa0d-587d95e058d4)
如图所示 无法统计拦截分布

### 重现步骤

如图所示 无法统计拦截分布 OpenResty也是最新版

### 期待的正确结果

更新最新版本后出现的这个问题

**Comment by wanghe-fit2cloud** [comment]
下载 v1.10.13-lts 安装包并解压，用最新的 1panel 二进制文件替换之前的 `/usr/local/bin/1panel` 文件，然后使用 `1pctl restart` 重启 1Panel 服务进行测试。

[安装包下载链接](https://resource.fit2cloud.com/1panel/package/stable/v1.10.13-lts/release/1panel-v1.10.13-lts-linux-amd64.tar.gz)

> 在操作之前，请备份 `/usr/local/bin/1panel` 文件。
